libs/schematics/package.json

```diff
@@ -0,0 +1,5 @@
+{
+	"name": "schematics",
```

Comment:
We could maybe have a nicer name for this, e.g.

`@ngx-nuts-and-bolts/schematics`

Just to be consistent with other packages, which all have this prefix